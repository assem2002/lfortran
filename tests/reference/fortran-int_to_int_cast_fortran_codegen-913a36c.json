{
    "basename": "fortran-int_to_int_cast_fortran_codegen-913a36c",
    "cmd": "lfortran --show-fortran --no-color {infile} -o {outfile}",
    "infile": "tests/int_to_int_cast_fortran_codegen.f90",
    "infile_hash": "4993e4b5ba22668cdfb76bf75a291c41d074af44ea3bd1bd143dbcd0",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "fortran-int_to_int_cast_fortran_codegen-913a36c.stdout",
    "stdout_hash": "fc977c8d595868fe252690e930c1fe3366582e434609a4a1fde96e32",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}