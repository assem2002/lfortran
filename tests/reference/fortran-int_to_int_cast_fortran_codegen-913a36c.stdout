program implicitinttointcastexample
implicit none
integer(4) :: integervalue1
integer(8) :: integervalue2
integervalue1 = 10
integervalue2 = int(integervalue1, kind=8)
print integervalue2 = int(integervalue1, kind=8)
, "Integer Value 1:", integervalue1
print print integervalue2 = int(integervalue1, kind=8)
, "Integer Value 1:", integervalue1
, "Integer Value 2 (Kind=8):",&
         integervalue2
end program implicitinttointcastexample
